docs/docs-content/clusters/edge/trusted-boot/key-management.md

```diff
@@ -0,0 +1,63 @@
+description: "Learn about how to manage the keys used in Trusted Boot."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["edge"]
+---
+
+Several key pairs are used in Trusted Boot during installer ISO generation, upgrade image generation, as well as
+installation. Each pair of keys serves a different purpose, is used during different stages of Edge artifact building
+and deployment, and need to be secured differently. This page discusses the different key pairs used by Trusted Boot and
+how to secure them.
+
+Secure key management is the foundation of all security benefits provided by Trusted Boot. All security provided by
+Trusted Boot assumes that your keys are securely stored. Ensure that you follow our recommendation to avoid compromising
+the security of your systems.
+
+## Platform Key (PK)
+
+The PK is at the top of the Secure Boot cryptographic key hierarchy. It's a key pair that has a private key and a public
+key.
+
+The private PK signs updates to the Key Exchange Key (KEK). The public PK is used to verify whether updates to the KEK
+are signed with the authentic private key and can be trusted.
+
+The private PK must be stowed away in a secure location **immediately** after being generated. You do not need the
+private key during EdgeForge, installation or deployment of your Edge devices. The public PK key is required during the
+EdgeForge build process so that it can be embedded into the Edge Installer ISO.
+
+## Key Exchange Key (KEK)
+
+Similar to how PK authenticates updates to the KEK key, the KEK authenticates updates to the Signature Database (DB). It
+also consists of pair that has a private key and a public key.
+
+The private KEK signs updates to the DB. When there are updates to the DB, the public KEK is used to verify that those
+updates are signed by the authentic private KEK.
+
+The private KEK must be stowed away in a secure location **immediately** after being generated. You do not need the
+private key during EdgeForge, installation or deployment of your Edge devices. The public KEK is required during the
+EdgeForge build process so that it can be embedded into the Edge Installer ISO.
+
+## Signature Database (DB) Key
+
+The signature database (DB) stores The db contains the authorized signatures or certificates for valid Unified Kernel
+Image (UKI) files. The DB key is also a pair that has a public key and a private key.
+
+The private DB key signs the UKI image in the Edge Installer ISO. The public DB key verifies the signature on the UKI
+image.
+
+Both the public and private DB keys should be stored securely in the build pipeline of your Edge artifacts, as they are
+needed during EdgeForge both during initial deployment and upgrades.
+
+## Platform Configuration Registers (PCR) Policy Key
+
+The PCR policy key is technically also a key pair, but you only see the private key after the key generation step. The
+public key is generated automatically during the EdgeForge process.
```

Comment:
```suggestion
public key is generated automatically during the EdgeForge process inside the generated artifacts.
```

Reply:
Maybe we should also expand here a bit: we can talk about how the PCR policy is used.

The PCR policy is used to seal the PCR banks: as such, after installation (which seals the PCR bank with the policy) only booting EFI files with those matching measurements will be able to unseal the same banks later on following reboots.